SU2_CFD/src/numerics/NEMO/convection/ausmplusup2.cpp

```diff
@@ -0,0 +1,576 @@
+/*!
+ * \file NEMO_slau.hpp
+ * \brief Declaration of numerics classes for the NEMO family of schemes,
+ *        including SLAU. The implementation is in NEMO.cpp.
+ * \author F. Palacios, T. Economon
+ * \version 7.0.3 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../../../include/numerics/NEMO/convection/ausmplusup2.hpp"
+
+CUpwAUSMPLUSUP2_NEMO::CUpwAUSMPLUSUP2_NEMO(unsigned short val_nDim, unsigned short val_nVar,
+                                           unsigned short val_nPrimVar,
+                                           unsigned short val_nPrimVarGrad,
+                                           CConfig *config): CNumerics (val_nDim, val_nVar, config){
```

Comment:
Is there nothing in common between AUSM and SLAU schemes for NEMO? even for the species part? :S I managed to re-use a ton for the ideal gas versions...